File: z_cs_py_yapilanlar/CLists.py
class CLists:
    def __init__(self):
        self.BarIndexList = []
        self.YonList = []
        self.SeviyeList = []
        self.SinyalList = []
        self.KarZararPuanList = []
        self.KarZararFiyatList = []
        self.KarZararFiyatYuzdeList = []
        self.KarAlList = []
        self.IzleyenStopList = []
        self.IslemSayisiList = []
        self.AlisSayisiList = []
        self.SatisSayisiList = []
        self.FlatSayisiList = []
        self.PassSayisiList = []
        self.KontratSayisiList = []
        self.VarlikAdedSayisiList = []
        self.KomisyonVarlikAdedSayisiList = []
        self.KomisyonIslemSayisiList = []
        self.KomisyonFiyatList = []
        self.KardaBarSayisiList = []
        self.ZarardaBarSayisiList = []
        self.BakiyePuanList = []
        self.BakiyeFiyatList = []
        self.GetiriPuanList = []
        self.GetiriFiyatList = []
        self.GetiriPuanYuzdeList = []
        self.GetiriFiyatYuzdeList = []
        self.BakiyePuanNetList = []
        self.BakiyeFiyatNetList = []
        self.GetiriPuanNetList = []
        self.GetiriFiyatNetList = []
        self.GetiriPuanYuzdeNetList = []
        self.GetiriFiyatYuzdeNetList = []
        self.GetiriKz = []
        self.GetiriKzNet = []
        self.GetiriKzSistem = []
        self.GetiriKzNetSistem = []
        self.EmirKomutList = []
        self.EmirStatusList = []

    def __del__(self):
        pass

    def initialize(self, Sistem):
        return self

    def reset(self, Sistem):
        self.create_lists(Sistem, Sistem.BarSayisi)
        return self

    def create_lists(self, Sistem, BarCount):
        self.BarIndexList = [0] * BarCount
        self.YonList = [""] * BarCount
        self.SeviyeList = [0.0] * BarCount
        self.SinyalList = [0.0] * BarCount
        self.KarZararPuanList = [0.0] * BarCount
        self.KarZararFiyatList = [0.0] * BarCount
        self.KarZararFiyatYuzdeList = [0.0] * BarCount
        self.KarAlList = [0.0] * BarCount
        self.IzleyenStopList = [0.0] * BarCount
        self.IslemSayisiList = [0.0] * BarCount
        self.AlisSayisiList = [0.0] * BarCount
        self.SatisSayisiList = [0.0] * BarCount
        self.FlatSayisiList = [0.0] * BarCount
        self.PassSayisiList = [0.0] * BarCount
        self.KontratSayisiList = [0.0] * BarCount
        self.VarlikAdedSayisiList = [0.0] * BarCount
        self.KomisyonVarlikAdedSayisiList = [0.0] * BarCount
        self.KomisyonIslemSayisiList = [0.0] * BarCount
        self.KomisyonFiyatList = [0.0] * BarCount
        self.KardaBarSayisiList = [0.0] * BarCount
        self.ZarardaBarSayisiList = [0.0] * BarCount
        self.BakiyePuanList = [0.0] * BarCount
        self.BakiyeFiyatList = [0.0] * BarCount
        self.GetiriPuanList = [0.0] * BarCount
        self.GetiriFiyatList = [0.0] * BarCount
        self.GetiriPuanYuzdeList = [0.0] * BarCount
        self.GetiriFiyatYuzdeList = [0.0] * BarCount
        self.BakiyePuanNetList = [0.0] * BarCount
        self.BakiyeFiyatNetList = [0.0] * BarCount
        self.GetiriPuanNetList = [0.0] * BarCount
        self.GetiriFiyatNetList = [0.0] * BarCount
        self.GetiriPuanYuzdeNetList = [0.0] * BarCount
        self.GetiriFiyatYuzdeNetList = [0.0] * BarCount
        self.GetiriKz = [0.0] * BarCount
        self.GetiriKzNet = [0.0] * BarCount
        self.GetiriKzSistem = [0.0] * BarCount
        self.GetiriKzNetSistem = [0.0] * BarCount
        self.EmirKomutList = [0.0] * BarCount
        self.EmirStatusList = [0.0] * BarCount
        return self

File: z_cs_py_yapilanlar/CLists.cs
public class CLists
{
    public List<string> YonList;
    public List<float> SeviyeList;
    public List<float> SinyalList;
    public List<float> KarZararPuanList;
    public List<float> KarZararFiyatList;
    public List<float> KarZararFiyatYuzdeList;
    public List<float> KarAlList;
    public List<float> IzleyenStopList;

    public List<float> IslemSayisiList;
    public List<float> AlisSayisiList;
    public List<float> SatisSayisiList;
    public List<float> FlatSayisiList;
    public List<float> PassSayisiList;

    public List<float> KontratSayisiList;
    public List<float> VarlikAdedSayisiList;
    public List<float> KomisyonVarlikAdedSayisiList;
    public List<float> KomisyonIslemSayisiList;
    public List<float> KomisyonFiyatList;
    public List<float> KardaBarSayisiList;
    public List<float> ZarardaBarSayisiList;

    public List<float> BarIndexList;

    public List<float> ZeroLevel { get; set; }

    public List<float> BakiyePuanList;
    public List<float> BakiyeFiyatList;
    public List<float> GetiriPuanList;
    public List<float> GetiriFiyatList;
    public List<float> GetiriPuanYuzdeList;
    public List<float> GetiriFiyatYuzdeList;

    public List<float> BakiyePuanNetList;
    public List<float> BakiyeFiyatNetList;
    public List<float> GetiriPuanNetList;
    public List<float> GetiriFiyatNetList;
    public List<float> GetiriPuanYuzdeNetList;
    public List<float> GetiriFiyatYuzdeNetList;

    public List<float> GetiriKz;
    public List<float> GetiriKzNet;

    public List<float> GetiriKzSistem;
    public List<float> GetiriKzNetSistem;

    public List<float> EmirKomutList { get; set; }
    public List<float> EmirStatusList { get; set; }

    ~CLists()
    {

    }

    public CLists()
    {

    }

    public CLists Initialize(dynamic Sistem)
    {
        return this;
    }

    public CLists Reset(dynamic Sistem)
    {
        return this;
    }

    public int CreateLists(dynamic Sistem, int BarCount)
    {
        int result = 0;

        YonList = new List<string>(new string[BarCount]);               // boş liste oluştur
        SeviyeList = Sistem.Liste(BarCount, 0);                         // boş liste oluştur
        SinyalList = Sistem.Liste(BarCount, 0);                         // boş liste oluştur
        KarZararPuanList = Sistem.Liste(BarCount, 0);                   // boş liste oluştur
        KarZararFiyatList = Sistem.Liste(BarCount, 0);                  // boş liste oluştur
        KarZararFiyatYuzdeList = Sistem.Liste(BarCount, 0);             // boş liste oluştur
        KarAlList = Sistem.Liste(BarCount, 0);                          // boş liste oluştur
        IzleyenStopList = Sistem.Liste(BarCount, 0);                    // boş liste oluştur

        IslemSayisiList = Sistem.Liste(BarCount, 0);                    // boş liste oluştur
        AlisSayisiList = Sistem.Liste(BarCount, 0);                     // boş liste oluştur
        SatisSayisiList = Sistem.Liste(BarCount, 0);                    // boş liste oluştur
        FlatSayisiList = Sistem.Liste(BarCount, 0);                     // boş liste oluştur
        PassSayisiList = Sistem.Liste(BarCount, 0);                     // boş liste oluştur

        KontratSayisiList = Sistem.Liste(BarCount, 0);                  // boş liste oluştur
        VarlikAdedSayisiList = Sistem.Liste(BarCount, 0);               // boş liste oluştur
        KomisyonVarlikAdedSayisiList = Sistem.Liste(BarCount, 0);       // boş liste oluştur
        KomisyonIslemSayisiList = Sistem.Liste(BarCount, 0);            // boş liste oluştur
        KomisyonFiyatList = Sistem.Liste(BarCount, 0);                  // boş liste oluştur
        KardaBarSayisiList = Sistem.Liste(BarCount, 0);                 // boş liste oluştur
        ZarardaBarSayisiList = Sistem.Liste(BarCount, 0);               // boş liste oluştur

        BarIndexList = Sistem.Liste(BarCount, 0);                       // boş liste oluştur

        ZeroLevel = Sistem.Liste(BarCount, 0);                          // boş liste oluştur

        BakiyePuanList = Sistem.Liste(BarCount, 0);                     // boş liste oluştur
        BakiyeFiyatList = Sistem.Liste(BarCount, 0);                    // boş liste oluştur
        GetiriPuanList = Sistem.Liste(BarCount, 0);                     // boş liste oluştur
        GetiriFiyatList = Sistem.Liste(BarCount, 0);                    // boş liste oluştur
        GetiriPuanYuzdeList = Sistem.Liste(BarCount, 0);                // boş liste oluştur
        GetiriFiyatYuzdeList = Sistem.Liste(BarCount, 0);               // boş liste oluştur

        BakiyePuanNetList = Sistem.Liste(BarCount, 0);                  // boş liste oluştur
        BakiyeFiyatNetList = Sistem.Liste(BarCount, 0);                 // boş liste oluştur
        GetiriPuanNetList = Sistem.Liste(BarCount, 0);                  // boş liste oluştur
        GetiriFiyatNetList = Sistem.Liste(BarCount, 0);                 // boş liste
        GetiriPuanYuzdeNetList = Sistem.Liste(BarCount, 0);             // boş liste oluştur
        GetiriFiyatYuzdeNetList = Sistem.Liste(BarCount, 0);            // boş liste oluştur

        GetiriKz = Sistem.Liste(BarCount, 0);                           // boş liste oluştur
        GetiriKzNet = Sistem.Liste(BarCount, 0);                        // boş liste oluştur

        GetiriKzSistem = Sistem.Liste(BarCount, 0);                     // boş liste oluştur
        GetiriKzNetSistem = Sistem.Liste(BarCount, 0);                  // boş liste oluştur

        EmirKomutList = Sistem.Liste(BarCount, 0);                      // boş liste oluştur
        EmirStatusList = Sistem.Liste(BarCount, 0);                     // boş liste oluştur

        return result;
    }
}